src/compile/axis/config.ts

```diff
@@ -12,6 +12,10 @@ export function getAxisConfig(
   // configTypes to loop, starting from higher precedence
   const configTypes = [
     ...(scaleType === 'band' ? ['axisBand'] : []),
+    ...(isContinuousToContinuous(scaleType) ? ['axisQuantitative'] : []),
```

Comment:
`isContinuousToContinuous` returns true for time and utc scales. Is this intended?

Reply:
Right, this is dumb.  We should fix this.

Reply:
@donghaoren realized this